content/browser/frame_host/navigation_request.cc

```diff
@@ -180,7 +180,7 @@ std::string AdaptUserAgentForURL(const std::string& user_agent, const GURL& url)
       url.DomainIs("nflxvideo.net")) {
     std::string::size_type start_pos = user_agent.find('(');
     std::string::size_type end_pos = user_agent.find(')');
-    result.replace(start_pos, end_pos - start_pos, std::string("X11; CrOS armv7l 10575.54.0"));
+    result.replace(start_pos + 1, end_pos - start_pos, std::string("X11; CrOS armv7l 10575.54.0"));
```

Comment:
So two problems - with start_pos + 1, we need to also update the len calculation somehow, otherwise we could replace one more char than intended (I guess squashing the `)`). But thinking about this, actually str::replace is not the correct way to implement this change. Because in the case that our replacement string is longer than `X11; Linux armv7l` we are actually going to truncate our replacement to the same length - ie `X11; CrOS armv7l 1` I think. Can you check this and fix it up?

Reply:
I have tested that just simply changing the calculation of start_pos to the char after the `(` seems to be properly replacing the UA, independently of the length. So after the last simple patch it would replace them as:
```
--- UA ( Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.92 Safari/537.36 )
--- UA ( Mozilla/5.0 (X11) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.92 Safari/537.36 )
```
and
```
--- UA ( Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.92 Safari/537.36 )
--- UA ( Mozilla/5.0 (X11; CrOS armv7l 10575.54.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.92 Safari/537.36 )
```